doc/source/reference/routines.strings.rst

```diff
@@ -0,0 +1,58 @@
+String universal functions
+==========================
+
+.. currentmodule:: numpy.strings
+
+.. module:: numpy.strings
+
+The `numpy.strings` module provides a set of universal functions operating
+on arrays of type `numpy.str_`, `numpy.bytes_` or `StringDType`.
+For example
+
+      >>> np.strings.add(["num", "doc"], ["py", "umentation"])
+      array(['numpy', 'documentation'], dtype='<U13')
```

Comment:
Maybe add a note here cross-referencing `np.char`, since not everything is implemented yet? It would also be good to note any differences between the `np.char` versions and the ufunc versions here.